Hi3Helper.Core/Lang/en.json

```diff
@@ -160,19 +160,19 @@
     "ListCol5": "Old Date",
     "ListCol6": "New Date",
     "ListCol7": "Status",
-    "Status1": "Click Check Update to check Caches update",
+    "Status1": "Click 'Check Update' to check Caches for updates",
```

Comment:
For syntax clarity, We can use escaped quote like ``\"``.

For example:
```JSON
    "Status1": "Click \"Check Update\" to check Caches for updates",
```

You can do the same for these lines:
14, 130, 167

Reply:
I see, I'll make the changes in the code for this. Thanks for the insight!